templates/scenarios/common/init-debug-vs-bot/teamsapp.local.yml

```diff
@@ -6,8 +6,8 @@ provision:
   - uses: teamsApp/create # Creates a Teams app
     with:
       name: bot # Teams app name
-    # Output: following environment variable will be persisted in current environment's .env file.
-    # TEAMS_APP_ID: the id of Teams app
+    writeToEnvironmentFile: # Write the information of installed dependencies into environment file for the specified environment variable(s).
```

Comment:
Why it is called `installed dependencies` here?

Reply:
Thanks, will fix it in another PR